### ISSUE TITLE
Rendering black bar when using UseModernWindowStyle

### ISSUE THREAD

**Issue by suddenlystorm** [Kinnara/ModernWpf#47]
I have created a sample app. The issue is that when  SizeToContent is set to "WidthAndHeight" and         ui:WindowHelper.UseModernWindowStyle is set to "True", there is a black bar at the right side of the app and underneath it, which resemble the former size of the window - without modern window style.

![2020-03-20](https://user-images.githubusercontent.com/62408712/77127143-1deba100-6a54-11ea-8a36-df0df9886c2e.png)

When UseModernWindowStyle is set to False, there is no problem whatsoever.

Thank you very much for all your work!

**Comment by Kinnara** [comment]
Thank you. This is a WPF bug and there are a few workarounds on [Stack Overflow](https://stackoverflow.com/questions/29207331/wpf-window-with-custom-chrome-has-unwanted-outline-on-right-and-bottom).

**Comment by mlaily** [comment]
@Kinnara Could the fix be applied directly by ModernWpf?

It would probably be this one: https://stackoverflow.com/questions/29207331/wpf-window-with-custom-chrome-has-unwanted-outline-on-right-and-bottom/51573942#51573942

This is what [MahApps does (more or less)](https://github.com/MahApps/MahApps.Metro/pull/2116/files), and it would be great if the user of the library didn't have to know how to do this themselves. What do you think?

I wouldn't mind submitting a PR, but I'm not sure about the best way to do this in your library...

**Comment by Kinnara** [comment]
@mlaily Good idea. I think we can add an attached property to `WindowHelper`. It would work similarly to the existing `FixMaximizedWindow` property.

**Comment by Kinnara** [comment]
Validated the exact scenario against current 1.x `master`; it no longer reproduces.

I built a disposable net8 WPF app with:

- `WindowHelper.UseModernWindowStyle=True`
- `SizeToContent=WidthAndHeight`
- `ResizeMode=CanResize`
- fixed 360×180 opaque content

The app showed the real HWND, waited for composition to settle, captured the complete window through native `PrintWindow`, and exited. The content and intended window border covered the full right and bottom edges; the reported black bands were absent. The disposable project was then removed and the worktree is clean.

Build command:

```text
dotnet build .\tools\repros\Issue47\Issue47Repro.csproj --configuration Release --no-restore
Build succeeded: 0 errors
```

This report targeted the pre-1.0 custom-chrome implementation, which has since been replaced by the audited 1.x WPF Fluent/custom shell. Closing as completed with no additional code change. If the artifact recurs with a current 1.x package, please open a fresh report including the Windows/.NET version, DPI, ResizeMode, dynamic content-size behavior, and a minimal project.
